TGM/src/main/java/network/warzone/tgm/parser/banner/BannerPatternsDeserializer.java

```diff
@@ -26,17 +27,25 @@
                     String in = element.getAsString();
                     String[] args = in.split(":");
                     DyeColor dyeColor = DyeColor.valueOf(Strings.getTechnicalName(args[0]));
-                    PatternType patternType = PatternType.valueOf(Strings.getTechnicalName(args[1]));
-                    if (patterns == null) patternType = PatternType.getByIdentifier(args[1]);
-                    patterns.add(new Pattern(dyeColor, patternType));
+                    PatternType patternType;
+                    if (EnumUtils.isValidEnum(PatternType.class, Strings.getTechnicalName(args[1]))) {
+                        patternType = PatternType.valueOf(Strings.getTechnicalName(args[1]));
```

Comment:
Nitpick but since `Strings.getTechnicalName(args[1])` appears twice, perhaps store it in a local variable instead